test/format/format.sh

```diff
@@ -368,7 +373,7 @@ format()
 		echo "$name: starting job in $dir"
 	fi
 
-	cmd="$format_binary -c "$config" -h "$dir" -1 $args quiet=1"
+	cmd="$pre_setting $format_binary -c "$config" -h "$dir" -1 $args quiet=1"
```

Comment:
An alternative approach might be "-b format-binary", which defaults to "./t" and can be anything you want, so:
```
format.sh -b "nice checksegv ./t"
```
would replace the format command with something else.

Reply:
I prefer this option.

Reply:
I also prefer this option. But I realised that "format_binary" is considered as an executable file in the script and using -b option to prefix something before binary will impact other exisiting validation checks. 

For example : 
If format_binary = "nice ./t", then the existing logic of checking of binary [-x $format_binary] would fail and changing directory to "$build_posix_directory" needs to changed.

I can still change the existing validation checks & to use -b option if @keithbostic & @lukech suggests. :-)

Reply:
I honestly don't feel strongly -- this makes it easier to replace the name of the format command and it occurred to me @lukech might find that useful, but it's at the cost of finding the last element of whatever is specified to the option.

Reply:
@raviprakashgiri29 in that case just choose whatever option that's easy to implement e.g. `format-prefix`.